client/src/components/favorite.js

```diff
@@ -5,38 +5,40 @@ import { API_URL } from "../constants";
 const Favorite = (props) => {
     const [trade, setTrade] = useState([]);
 
-    const loadTrade = () => {
```

Comment:
try and use `async/await` syntax here instead of using promise chains. I'm suggesting this since you used `async/await` in `logout-files.js`. You should use a consistent approach throughout your application.